spring-cloud-dataflow-admin-mesos/src/main/java/org/springframework/cloud/dataflow/module/deployer/marathon/MarathonModuleDeployer.java

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.dataflow.module.deployer.marathon;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+import mesosphere.marathon.client.Marathon;
+import mesosphere.marathon.client.MarathonClient;
+import mesosphere.marathon.client.model.v2.App;
+import mesosphere.marathon.client.model.v2.Container;
+import mesosphere.marathon.client.model.v2.Docker;
+import mesosphere.marathon.client.model.v2.Port;
+import mesosphere.marathon.client.model.v2.Task;
+import mesosphere.marathon.client.utils.MarathonException;
+
+import org.springframework.cloud.dataflow.core.ModuleDeploymentId;
+import org.springframework.cloud.dataflow.core.ModuleDeploymentRequest;
+import org.springframework.cloud.dataflow.module.ModuleStatus;
+import org.springframework.cloud.dataflow.module.deployer.ModuleArgumentQualifier;
+import org.springframework.cloud.dataflow.module.deployer.ModuleDeployer;
+
+/**
+ * A ModuleDeployer implementation for deploying modules as applications on Marathon, using the
+ * ModuleLauncher Docker image.
+ *
+ * @author Eric Bottard
+ */
+public class MarathonModuleDeployer implements ModuleDeployer {
+
+	/**
+	 * The name of the ENV variable that is added to apps to identify Spring Cloud Data Flow modules.
+	 */
+	private static final String SPRING_CLOUD_DATAFLOW_MODULE = "SPRING_CLOUD_DATAFLOW_MODULE";
+
+	public static final String CONNECTOR_DEPENDENCY = "org.springframework.cloud:spring-cloud-marathon-connector:1.0.0.BUILD-SNAPSHOT";
```

Comment:
Can we get the connector dependency version via external properties (possibly from admin.yml)?